addons/block-palette-icons/userscript.js

```diff
@@ -0,0 +1,34 @@
+//When the page loads add the icons.
+export default async function ({ addon, global, console }) {
+  while (true) {
+    const tabs = await addon.tab.waitForElement(".scratchCategoryMenu", {
+      markAsSeen: true,
+    });
+
```

Comment:
why?

Reply:
So that when adding another extension it keeps the icons.

Reply:
Otherwise adding the pen extension, or anything else just removes it.

Reply:
huh